pkg/cli/environment.go

```diff
@@ -77,6 +83,8 @@ func (s *EnvSettings) AddFlags(fs *pflag.FlagSet) {
 	fs.StringVarP(&s.namespace, "namespace", "n", s.namespace, "namespace scope for this request")
 	fs.StringVar(&s.KubeConfig, "kubeconfig", "", "path to the kubeconfig file")
 	fs.StringVar(&s.KubeContext, "kube-context", s.KubeContext, "name of the kubeconfig context to use")
+	fs.StringVar(&s.Token, "token", s.Token, "bearer token used for authentication")
+	fs.StringVar(&s.APIServer, "server", s.APIServer, "the address and the port for the Kubernetes API server")
```

Comment:
kubectl has the `-s` shortflag for `--server`.  Do we want to do the same?

Reply:
This would prevent other subcommands from using `-s` as a shortflag, right? If so that would block other existing shortflags like `helm create -s/--starter`

Reply:
Ouch! Nice save.  Thanks!